### ISSUE TITLE
Add ObjectBox and Sembast

### ISSUE THREAD

**Issue by ueman** [Manuelbaun/serializer_libs_comparison#1]
I guess it would be nice if the comparison includes 
- [ObjectBox](https://pub.dev/packages/objectbox) 
- [Sembast](https://pub.dev/packages/sembast) 

They aren't strictly speaking a serializer, however they are solving more or less the same use case as Hive and Isar.

**Comment by Manuelbaun** [comment]
@ueman I will look at them. 

Although as far as I know, ObjectBox is using flatbuffers under the hook, I can look at how Sembast is solving the serializing part and compare them to how Hive or Flatbuffers perform.

**Comment by ueman** [comment]
That's interesting I didn't know that about ObjectBox.

**Comment by ueman** [comment]
Looks like Sembast uses JSON textfiles:
https://github.com/tekartik/sembast.dart/blob/master/sembast/doc/storage_format.md

**Comment by Manuelbaun** [comment]
Oh I see, interesting.

So, essentially the serializing speed depends on how fast someone can serialize JSON. I would assume, It would be much slower?

**Comment by ueman** [comment]
There's just one way to find out.

Well I would guess that it's not that slow because the file is append only. However there are compactions every now and then to reduce the file size which could cause seemingly random slow downs.

**Comment by Manuelbaun** [comment]
> There's just one way to find out.

absolutely. I will have to take a look since I only use the serialization of the libs to compare, not actually the write to the database.
